content/engine/security/rootless.md

```diff
@@ -167,7 +170,11 @@ testuser:231072:65536
 > If the system-wide Docker daemon is already running, consider disabling it:
 >```console
 >$ sudo systemctl disable --now docker.service docker.socket
+>$ sudo systemctl stop docker.service docker.socket
```

Comment:
Why doesn't `disable --now` work?

Reply:
I suppose it's just the docker socket is still present on the system.  `/var/run/docker.socket` needs to be forcefully removed in this step rather than simply `stop` on the service and socket.  This allows the user to run `dockerd-rootless-setuptool.sh install` without the `--force` parameter.